webhooks-extension/endpoints/webhook_test.go

```diff
@@ -96,6 +99,34 @@ func testGitHubSource(expectedName string, expectedOwnerAndRepo string, expected
 	}
 }
 
+func testGithubSourceReleaseNameTooLong(r *Resource, t *testing.T) {
+
+	runNs := "test"
+
+	data := webhook{
+		Name:                 "name1",
+		Namespace:            runNs,
+		GitRepositoryURL:     "https://github.com/owner/repo.git",
+		AccessTokenRef:       "token1",
+		Pipeline:             "pipeline1",
+		RegistrySecret:       "registrysecret1",
+		HelmSecret:           "helmsecret1",
+		RepositorySecretName: "secretName1",
+		ReleaseName:          "1234567891234567891234567891234567891234567891234567891234567890", // 0 brings us to 64 char
+	}
+
+	b, _ := json.Marshal(data)
+	httpReq := dummyHTTPRequest("POST", "http://wwww.dummy.com:8080/webhooks-extension/webhook/", bytes.NewBuffer(b))
+	req := dummyRestfulRequest(httpReq, "", "")
+	httpWriter := httptest.NewRecorder()
+	resp := dummyRestfulResponse(httpWriter)
+	r.createWebhook(req, resp)
```

Comment:
Please use the helper `createWebhook(webhook, resource)` function, instead of re-writing the code to create webhooks with dummy http requests & responses. This will minimize duplicate code.

Since your tests use the `resp`, you can update the `createWebhook` function to return the `resp`, and the tests that don't need it don't have to worry about what's being returned.